src/main/java/com/atherys/towns/model/entity/Town.java

```diff
@@ -220,4 +224,12 @@ public void setBank(UUID bank) {
         this.bank = bank;
     }
 
+    public LocalDateTime getLastRaidCreationDate() {
```

Comment:
Do we want to leverage the `Optional<LocalDateTime>` in our plugin as well?

Reply:
Going to look into this..